crates/sozo/ops/src/migration/mod.rs

```diff
@@ -105,38 +107,20 @@ where
     let mut strategy = prepare_migration(&target_dir, diff, name.clone(), world_address, &ui)?;
     let world_address = strategy.world_address().expect("world address must exist");
 
-    if dry_run {
+    let migration_output = if dry_run {
         print_strategy(&ui, account.provider(), &strategy).await;
-
-        update_manifests_and_abis(
-            ws,
-            local_manifest,
-            &profile_dir,
-            &profile_name,
-            MigrationOutput { world_address, ..Default::default() },
-            name.as_ref(),
-        )
-        .await?;
+        MigrationOutput { world_address, ..Default::default() }
```

Comment:
Isn't this PR depending on #1790? :) Or should we merge this first @lambda-0x?

Reply:
i think both are independent